flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
so why this:
![image](https://github.com/flathub/flathub/assets/114441/78f1f77b-fa14-4f72-bb3a-467586ca9aaa)

Reply:
Only building x64 for the demo right now, might expand architectures if this flatpak gets enough traction. What yml changes are needed to install x64 and aarch64 binaries?